routes/v1.js

```diff
@@ -60,30 +60,30 @@ var postProc = {
  * @param {object} peliasConfig
  */
 function addRoutes(app, peliasConfig) {
+  const esclient = require('elasticsearch').Client(peliasConfig.esclient);
```

Comment:
nit pick: could have this `require` at the top of the file with the other `require` statements?